Missing include netinet/in.h while compiling openssh_fixture.c on FreeBSD

Building libssh2 on FreeBSD fails while compiling openssh_fixture.c, this is because netinet/in.h is not included.

Here after the error message:

```
[ 72%] Building C object tests/CMakeFiles/openssh_fixture.dir/openssh_fixture.c.o
/home/tmow/Software/libssh2/libssh2.antenore/tests/openssh_fixture.c: In function 'open_socket_to_container':
/home/tmow/Software/libssh2/libssh2.antenore/tests/openssh_fixture.c:229:40: error: storage size of 'sin' isn't known
                     struct sockaddr_in sin;
                                        ^
/home/tmow/Software/libssh2/libssh2.antenore/tests/openssh_fixture.c:236:40: error: invalid application of 'sizeof' to incomplete type 'struct sockaddr_in'
                                 sizeof(struct sockaddr_in)) == 0) {
                                        ^
/home/tmow/Software/libssh2/libssh2.antenore/tests/openssh_fixture.c:229:40: warning: unused variable 'sin' [-Wunused-variable]
                     struct sockaddr_in sin;
                                        ^
*** Error code 1
```

I'm fixing this myself and I'll submit a PR.

--- a/tests/libssh2_config_cmake.h.in
+++ b/tests/libssh2_config_cmake.h.in
@@ -39,6 +39,7 @@
 #cmakedefine HAVE_INTTYPES_H
 #cmakedefine HAVE_SYS_SOCKET_H
 #cmakedefine HAVE_ARPA_INET_H
+#cmakedefine HAVE_NETINET_IN_H
 #cmakedefine HAVE_WINDOWS_H
 #cmakedefine HAVE_WINSOCK2_H
 #cmakedefine HAVE_SNPRINTF

--- a/tests/openssh_fixture.c
+++ b/tests/openssh_fixture.c
@@ -47,6 +47,9 @@
 #ifdef HAVE_ARPA_INET_H
 #include <arpa/inet.h>
 #endif
+#ifdef HAVE_NETINET_IN_H
+#include <netinet/in.h>
+#endif
 #ifdef HAVE_UNISTD_H
 #include <unistd.h>
 #endif

